Jenkinsfile

```diff
@@ -364,15 +364,20 @@ pipeline {
       }
     }
     stage('Deploy') {
+      when { branch 'master' }
```

Comment:
why did you remove the beforeAgent line?

Reply:
I put it back.